kalliope/core/ConfigurationManager/utils.py

```diff
@@ -0,0 +1,10 @@
+import inspect
+import os
+
+
+def get_root_kalliope_path():
+    # here we are in /an/unknown/path/kalliope/core/ConfigurationManager
+    current_script_path = os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe())))
+    # get parent dir. Now we are in /an/unknown/path/kalliope
+    kalliope_root_path = os.path.normpath(current_script_path + os.sep + os.pardir + os.sep + os.pardir)
+    return kalliope_root_path
```

Comment:
This function can be only used, as designed, by a script placed in ConfigurationManager folder. If another script call it, the path is not the same.